packages/@react-spectrum/icon/src/Illustration.tsx

```diff
@@ -57,13 +57,17 @@ export function Illustration(props: IllustrationProps) {
     ariaHidden = undefined;
   }
 
+  let width = children.props.width || '32';
+  let height = children.props.height || '32';
+
   return React.cloneElement(children, {
     ...filterDOMProps(otherProps),
     ...styleProps,
     focusable: 'false',
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledby,
     'aria-hidden': ariaHidden,
-    role: hasLabel ? 'img' : undefined
+    role: hasLabel ? 'img' : undefined,
+    viewBox: `0 0 ${width} ${height}`
```

Comment:
I believe svgs need a viewBox to fill the width of their parent container. This is the only way I could figure out how to get it working correctly without breaking existing illustrations (IllustratedMessage).

Reply:
This technically means all of the illustrations now scale, even within an illustrated message. Is that what we want or do we want to keep this more limited to File/Folder for now? If so, you could just apply the width/height/viewBox attributes within those specific SVGs.

Reply:
I think it may be safer to set height/width/viewBox on just those for now, like you said. Let's go that route.